src/show/pages/private/MyProjects/index.js

```diff
@@ -0,0 +1,132 @@
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+import { Fragment, useState, useEffect } from 'react';
+import { FaPencilAlt, FaTrash } from 'react-icons/fa';
+
+import {
+  EDIT_PROJECT_PATH,
+  PROJECT_PATH
+} from '../../../../process/routes/paths';
+import { Button, ProgressBar, Modal } from '../../../components';
+
+const MyProjects = () => {
+  const [projectId, setProjectId] = useState(null);
+  const [myProjects, setMyProjects] = useState([]);
+  const [projectModal, setProjectModal] = useState(false);
+
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    axios
+      .get(`http://localhost:3000/v1/projects/my_projects`)
+      .then(res => setMyProjects(res.data))
+      .catch(err => err);
+  }, []);
+
+  const deleteProject = e => {
+    e?.preventDefault();
+    axios
+      .delete(`http://localhost:3000/v1/projects/${projectId}`)
+      .then(() => {
+        const filterProject = object => projectId !== object.id;
+        const filteredProjects = myProjects.filter(filterProject);
+        setProjectModal(false);
+        setMyProjects(filteredProjects);
+      })
+      .catch(err => err);
+  };
+  const getProjectId = project => {
```

Comment:
Empty line before this for readability and rename to something better.
The action that you do are opening a delete modal and saving the project id.